tests/instruments/teledyne/test_teledyneMAUI_with_device.py

```diff
@@ -309,6 +309,12 @@ def test_trigger_setup(self, resetted_instrument):
         resetted_instrument.trigger_setup(**expected)
         assert resetted_instrument.trigger == expected
 
+    def test_vbs(self, resetted_instrument):
+        """Test reading and writing values through the VBS interface."""
+        resetted_instrument.vbs_write("app.Acquisition.C1.VerScale=0.01")
+        val = resetted_instrument.vbs_ask("app.Acquisition.C1.VerScale")
```

Comment:
I had misunderstood your comment. Yes, you could create an instrument property for e.g. vertical scale like that.

But I don't think it has any benefit over the regular GPIB command `C1:VDIV`, and I don't see the need to add extra property / replace the regular ones.

I just wanted to provide uses with the option to use VBS commands in case there are functions that are otherwise not available.

Reply:
> Yes, you could create an instrument property for e.g. vertical scale like that.

Ok, that was my question out of curiosity, thinking about use cases not otherwise achievable (see your last sentence).
It's really great, that you can just use the "VBS magic" in normal properties!